RawSpeed/Cr2Decoder.cpp

```diff
@@ -190,6 +191,104 @@ void Cr2Decoder::decodeMetaDataInternal(CameraMetaData *meta) {
   if (mRootIFD->hasEntryRecursive(ISOSPEEDRATINGS))
     iso = mRootIFD->getEntryRecursive(ISOSPEEDRATINGS)->getInt();
 
+  // Fetch the white balance
+  if (mRootIFD->hasEntryRecursive(CANONCOLORDATA)) {
+    TiffEntry *color_data = mRootIFD->getEntryRecursive(CANONCOLORDATA);
+
+    // this entry is a big table, and different cameras store used WB in
+    // different parts, so find the offset
+    int offset = 0;
+    switch (color_data->count)
```

Comment:
Please use this value as a *default* value, add a hint that allow us to override the value like this

```<Hint name="wb_offset" value="72"/>```

That way, we can add support for new cameras without modifying code.